common/protob/messages-ripple.proto

```diff
@@ -29,23 +29,288 @@ message RippleAddress {
  * @next RippleSignedTx
  */
 message RippleSignTx {
-	repeated uint32 address_n = 1;				// BIP-32 path. For compatibility with other wallets, must be m/44'/144'/index'
-	optional uint64 fee = 2;					// fee (in drops) for the transaction
-	optional uint32 flags = 3;					// transaction flags
-	optional uint32 sequence = 4;				// transaction sequence number
-	optional uint32 last_ledger_sequence = 5;	// see https://developers.ripple.com/reliable-transaction-submission.html#lastledgersequence
-	optional RipplePayment payment = 6;			// Payment transaction type
+	// Common fields
+	repeated uint32 address_n = 1; // BIP-32 path. For compatibility with other wallets, must be m/44'/144'/index'
+	optional uint64 fee = 2; // fee (in drops) for the transaction
+	optional uint32 sequence = 3; // transaction sequence number
```

Comment:
You can't change the existing fields, only add the new ones. This will break compatibility. Please remove the changes that modify the existing fields.

Reply:
Is it also necessary to keep the order/"protobuf tag" of existing fields, or just the name/type?

Reply:
Order is also important

Reply:
Ok, I've changed it to match the previous order (looks a bit weird, but I guess there is not much we can do about that). Now the old tests pass :)